README.md

```diff
@@ -43,6 +43,16 @@ both footprint and startup time are behaved pretty well.
 
 ```
 
+### Transaction reattachment(WIP)
```

Comment:
Don't mention WIP here. Instead, you can set label "WIP" in pull request.

Reply:
I used WIP here is to indicate the whole process of reattachment has not completed.
Should I mention it in `README.md`?

Reply:
Frequent iterations are crucial to development process. We shall concentrate on the functionality, and the documentation is just an entry to them.